lib/routes/account.js

```diff
@@ -579,7 +599,8 @@ module.exports = function (
                           // value; Its stable, random, and will change on
                           // account reset.
                           wrapKb: account.wrapWrapKb,
-                          emailVerified: true
+                          emailVerified: true,
+                          tokenVerificationId: tokenVerificationId
```

Comment:
> These tokens will never get verified because we never send an email, right?

Seems that way. What happens then? Is there a way to say "this session cannot be used for keys" without ever creating a corresponding verification token?